sys/android-architecture.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
+	echo "Please enter an android architecture..."
+fi
+
+
+case "$1" in
+	aarch64)
+		cd `dirname $PWD/$0`
+		export CFLAGS="-O3"
```

Comment:
cflags should be the same for ALL archs, not different between them.. also should be overridable by the user